src/global/project/manifest.rs

```diff
@@ -431,40 +431,52 @@ impl Manifest {
 #[derive(Debug, Clone, Serialize, Deserialize, Hash, PartialEq, Eq)]
 pub struct Mapping {
     exposed_name: ExposedName,
-    executable_name: String,
+    executable_relname: String,
```

Comment:
What is the reason to change to `relname`? And what does it mean to you?

Reply:
I'm not a huge fan of this name too. It's a relative path but with no extension, maybe you'll have an idea for a better name.
I want to be able to access both values executable_relname and executable_name  that's why I changed this attribute name.